app_with_external_db.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        RAILS_ENV: production
+    network_mode: host
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails db:prepare && bundle exec rails s -b '0.0.0.0'"
```

Comment:
@ksnitsky is it ok for us to use `bundle exec rails db:prepare` right here and do not create an additional service?